crypto/threads_pthread.c

```diff
@@ -57,10 +57,13 @@ CRYPTO_RWLOCK *CRYPTO_THREAD_lock_new(void)
      * We don't use recursive mutexes, but try to catch errors if we do.
      */
     pthread_mutexattr_init(&attr);
-#  if !defined(NDEBUG) && !defined(OPENSSL_NO_MUTEX_ERRORCHECK)
+#  if ! (defined (__TANDEM) || defined (_SPT_MODEL_))
```

Comment:
Style nit: delete the space after the !
Alternatively apply De Morgan's: `!defined (__TANDEM) && !defined (_SPT_MODEL_)` to shorten it.

Reply:
@rsbeckerca could you please apply paulidale's suggestion to shorten the condition?

Reply:
> 
> 
> @rsbeckerca could you please apply paulidale's suggestion to shorten the condition?

I went with @paulidale's second option to shorten it.